spec.html

```diff
@@ -63,6 +63,74 @@ <h1>Contributing to this Proposal</h1>
   </ul>
 </div>
 
+<emu-clause id="sec-well-known-intrinsic-objects">
+  <h1>Well-Known Intrinsic Objects</h1>
+
+  <emu-table id="table-7" caption="Well-Known Intrinsic Objects">
+    <table>
+      <tr>
+        <th>
+          Intrinsic Name
+        </th>
+        <th>
+          Global Name
+        </th>
+        <th>
+          ECMAScript Language Association
+        </th>
+      </tr>
+      <tr>
+        <td>
+          <ins>%AsyncIterator%</ins>
+        </td>
+        <td>
+          <ins>`AsyncIterator`</ins>
+        </td>
+        <td>
+          <ins>The `AsyncIterator` constructor (<emu-xref href="#sec-asynciterator-constructor"></emu-xref>)</ins>
+        </td>
+      </tr>
+      <tr>
+        <td>
+          %AsyncIteratorPrototype%
```

Comment:
```suggestion
          %AsyncIterator.prototype%
```

Reply:
The [`%AsyncIteratorPrototype%`](https://tc39.es/ecma262/#sec-asynciteratorprototype) intrinsic is specified in <https://tc39.es/ecma262/#sec-well-known-intrinsic-objects>, which is why it doesn’t have `<ins>`.